pos_order_mgmt/static/src/js/widgets.js

```diff
@@ -210,12 +210,24 @@ odoo.define('pos_order_mgmt.widgets', function (require) {
             order.trigger('change');
             this.pos.get('orders').add(order);
             this.pos.set('selectedOrder', order);
+            order.get_orderlines().forEach(function (orderline) {
```

Comment:
I feel like this isn't the right method to do it. Why not to put it in `_prepare_order_from_order_data` or in `_prepare_orderlines_from_order_data`?

Reply:
Because in that case it would affect other actions like `copy`.
IMO we only need this in this specific action. If you duplicate an order you expect to fill new lots.

Reply:
@chienandalu I moved it here: https://github.com/OCA/pos/pull/467/commits/ff281e4811f614b8a41c158fd0edc7ff789a255e

I agree it's better here ^^